main.js

```diff
@@ -0,0 +1,56 @@
+/*task 6*/
+let user = 'John Doe';
+let student = 'Irina';
+
+console.log(user);
+console.log(student);
+
+user = student; /*Irina*/
+
+console.log(user);
+
+/*task 7*/
+let test = 1;
+
+test++; /*2*/
+test = test + '1'; /*21*/
+
+console.log(test);
+
+test = test - 1; /*20*/
+
+console.log(test);
+
+if (test === 20) {
+    console.log(true);
+} else {
+    console.log(false);
+}
```

Comment:
What is it for?
Google the ways to convert the value of variable to a boolean.

Reply:
I think I  understood all and corrected it.